src/main/java/edu/kit/ipd/crowdcontrol/objectservice/ExperimentController.java

```diff
@@ -0,0 +1,120 @@
+package edu.kit.ipd.crowdcontrol.objectservice;
+
+import edu.kit.ipd.crowdcontrol.objectservice.crowdworking.PlatformManager;
+import edu.kit.ipd.crowdcontrol.objectservice.crowdworking.TaskOperationException;
+import edu.kit.ipd.crowdcontrol.objectservice.event.ChangeEvent;
+import edu.kit.ipd.crowdcontrol.objectservice.event.Event;
+import edu.kit.ipd.crowdcontrol.objectservice.event.EventManager;
+import edu.kit.ipd.crowdcontrol.objectservice.event.EventObservable;
+import edu.kit.ipd.crowdcontrol.objectservice.proto.Experiment;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import rx.Observable;
+import rx.Observer;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by lucaskrauss at 05.02.2016
+ */
+public class ExperimentController implements Observer<Event<Experiment>> {
+
+    private final Logger log = LogManager.getLogger(ExperimentController.class);
+    private final Observable<Event<Experiment>> observable = EventManager.EXPERIMENT_CREATE.getObservable();
+    private final EventObservable<ChangeEvent<Experiment>> endExpObservable = EventManager.EXPERIMENT_CHANGE;
+    private final PlatformManager platformManager;
+
+    public ExperimentController(PlatformManager manager){
+        platformManager = manager;
+        observable.subscribe();
```

Comment:
have you tested this? this is never going to work you subscribe nothing you will never get called.